client/dashboard/components/section-header/index.tsx

```diff
@@ -48,15 +49,17 @@ export const SectionHeader = ( {
 						) }
 					</VStack>
 
-					{ /* The wrapper is always needed for view transitions. */ }
-					<ButtonStack
-						justify="flex-end"
-						expanded={ false }
-						alignment="flex-start"
-						className="dashboard-section-header__actions"
-					>
-						{ actions }
-					</ButtonStack>
+					{ !! actions && (
```

Comment:
I'm making this conditional otherwise, there is an empty gap for containers with no actions. This doesn't impact the transition animation we currently have, and likely from when we had more advanced transitions. We can come back to fix this in the future if we want.